test/12_nullablearray.jl

```diff
@@ -970,4 +970,27 @@ for ordered in (false, true)
     end
 end
 
+# Test vcat with nulls
+@test isequal(vcat(NullableCategoricalArray(["a","b"],[false,true]),
+           NullableCategoricalArray(["b","a"],[true,false])),
```

Comment:
You can use `==` instead of  `isequal` since I overwrote it at the top of the file for simplicity.